modis_tools/models.py

```diff
@@ -94,9 +99,9 @@ class Granule(ApiEntry):
     cloud_cover: Optional[str] = None
     collection_concept_id: str
     day_night_flag: Optional[str] = None
-    granule_size: float
+    # granule_size: float
```

Comment:
I observed that it (`test.py`) broke when this was un-commented, but making the field optional worked. Proposal to do this b/c it's backwards compatible if there are cases where the `granule_size` field is provided by the API and then used downstream
```suggestion
    granule_size: Optional[float] = None
```